src/element/newElement.ts

```diff
@@ -63,15 +64,7 @@ export function newTextElement(
 }
 
 export function duplicateElement(element: ReturnType<typeof newElement>) {
-  const copy = {
-    ...element,
-  };
-  if ("points" in copy) {
-    copy.points = Array.isArray(element.points)
-      ? JSON.parse(JSON.stringify(element.points))
-      : element.points;
-  }
-
+  const copy = klona(element);
   delete copy.shape;
```

Comment:
Not a big deal but this is going to be less efficient than the current setup. Shape is a non trivial object so we’ll deep copy it to then delete it.

This is fine since the copy operation is not done very often and copying the shapes object is also not that big in the grand scheme of things.

Reply:
> > I'm actually reconsidering the use of `klona` :) because it turns out it's not working correctly. It copies prototype properties over onto the object. I'm in the process of creating an issue over in the repo, and currently it won't cause issues for us (AFAIK), so duno.
> 
> Lodash's https://lodash.com/docs/4.17.15#cloneDeep might work, nah?

Yea, but that's an overkill IMO. It'll be even less performant, and it'll also likely add a lot to our bundle (even considering tree shaking).